cloudinit/net/sysconfig.py

```diff
@@ -697,7 +709,16 @@ def _render_vlan_interfaces(cls, network_state, iface_contents, flavor):
                 iface_cfg['ETHERDEVICE'] = iface_name[:iface_name.rfind('.')]
             else:
                 iface_cfg['VLAN'] = True
-                iface_cfg['PHYSDEV'] = iface_name[:iface_name.rfind('.')]
+                iface_cfg.kind = 'vlan'
+
+                rdev = iface['vlan-raw-device']
+                supported = _supported_vlan_names(rdev, iface['vlan_id'])
+                if iface_name not in supported:
+                    LOG.warning(
+                        "Name '%s' for vlan '%s' is not officially supported"
+                        "by RHEL. Supported: %s",
```

Comment:
This message will be emitted on any non-SUSE system that uses sysconfig (`self.flavor` defaults to "rhel"). I don't think the text appearing in the log is a major problem (though I could be convinced otherwise), but it doesn't seem ideal to emit a _warning_ on all such systems: this may not be a problem at all on some systems.

What do you think about dropping this down to INFO?

(If you wanted it to be a WARNING on RHEL, that could come as a follow-up; we don't have access to a distro object here (AFAICT), though, so I'm not sure what the best way to go about that would be.)